b2sdk/transfer/inbound/download_manager.py

```diff
@@ -42,18 +42,20 @@ class DownloadManager(metaclass=B2TraceMetaAbstract):
     MIN_CHUNK_SIZE = 8192  # ~1MB file will show ~1% progress increment
     MAX_CHUNK_SIZE = 1024**2
 
-    def __init__(self, services):
+    def __init__(self, services, max_download_workers=10):
```

Comment:
default is set to 10 in Services, in B2Api and in DownloadManager. Is this right?

Reply:
Yes, I wanted to be consistent with `max_upload_workers` and `max_copy_workers`. In general, I believe the default should be rather `None` (let the ThreadPool implementation choose appropriate default), but I'm not sure if I can change this without breaking public API compatibility.

Reply:
lets not keep consistency with our past bad practices

Reply:
those classes are not members of the public interface so you can change them at will